.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please remove `.vscode` and all related files from the repository by adding it to the `.gitignore`-file and using `git rm --cached -r .vscode`.